--- flix-engine-installer-linux-8.0.7.1_DEMO/.flix-engine-installation-files/flixphp/target.mk	2006-12-28 19:46:51.078812726 +0200
+++ flix-engine-installer-linux-8.0.7.1_DEMO/.flix-engine-installation-files/flixphp/target.mk	2006-12-28 19:47:59.117480649 +0200
@@ -25,12 +25,12 @@
 PHPINST := $(shell php-config --extension-dir)
 $(PHPWRAP_SO): $(PHPWRAP_C)
 	$(CC) $(CFLAGS) $(PHPINCS) -shared $(PHPWRAP_C) \
-              -lflixengine2 -o $(PHPWRAP_SO) 
+             -L$(LIBDIR) -lflixengine2 -o $(PHPWRAP_SO) 
 
 install:
-	mkdir -p $(PHPINST)
-	install -p $(PHPWRAP_SO) $(PHPINST)
-	install -p $(PHPWRAP_PHP) $(PHPINST)/..
+	mkdir -p $(DESTDIR)$(PHPINST)
+	install -p $(PHPWRAP_SO) $(DESTDIR)$(PHPINST)
+	install -p $(PHPWRAP_PHP) $(DESTDIR)/usr/lib
     
 clean:
 	$(RM) -f $(PHPWRAP_SO)
